lib/span-data.js

```diff
@@ -82,6 +92,42 @@ SpanData.prototype.close = function() {
 };
 
 /**
+ * Trace API expects stack frames to be a JSON string with the following
+ * structure:
+ * STACK_TRACE := { "stack_frame" : [ FRAMES ] }
+ * FRAMES := { "class_name" : CLASS_NAME, "file_name" : FILE_NAME,
+ *             "line_number" : LINE_NUMBER, "method_name" : METHOD_NAME }*
```

Comment:
Nit: could we include columnNumber in this graphic as it is a member of the class.